mxcube3/ui/components/SampleQueue/QueueControl.js

```diff
@@ -46,7 +46,8 @@ export default class QueueControl extends React.Component {
           { text: 'New Sample', class: 'btn-primary', action: this.showForm, key: 1 },
         ],
         LastSample: [
-          { text: 'Finish', class: 'btn-primary', action: this.nextSample, key: 1 },
+          { text: 'Unmount and terminate the queue', class: 'btn-primary',
```

Comment:
Oh, this is great, un-mount is indeed more descriptive. I'm not so certain about "terminate queue" though, I think Unmount or "Unmount and Finish" or "Finish and Unmount" even better.

Reply:
Our people was confuse for the 'finish', they understood finish as 'finish the rest of the queue' instead of leaving where it is

Reply:
Terminate is a bit socking, but we could not figure out a better text

Reply:
Yes finish is not a good word.

What the finish button really does, is to unmount the last sample in a queue that was just executing. The queue is not running when the user presses this button. The button is there to prevent users to forget samples on the gonio.

There are normally not any other samples in the queue when this button is presented so there should be no scenario for 'finish the rest of the queue'.

I agree that finish is not a good term, but I don't think adding terminate in anyway would make it any better ?

Reply:
The problem is that if there are some un-executed tasks in the current sample, finish could be understood as 'finish the remaining tasks'.

So, maybe simply 'Unmount sample'?

Reply:
Why not "Simply unmount" or "Unmount only". It might be more explicit if you do it the other way arround: "Unmount and countinue" and "Unmount".